### PR TITLE
Changed when environment takes effects 

### PR DESCRIPTION
**Pull request by Mark7625**
With the old system if you was near another Environment and your camera angle was within that location sometimes it would change the Environment  based on where your camera was in, this change only changes once the player is within this range

https://i.imgur.com/ltHYzux.gif

### CODE REVIEW

**Comment by Mark7625** [comment]
Please feel free to discuss any reasons why it was based on the camera and not the player

**Review by sosodev — CHANGES_REQUESTED** [review]

File: src/main/java/rs117/hd/environments/EnvironmentManager.java [hunk 1]
```diff
@@ -138,14 +138,14 @@
 
 	public void update()
 	{
-		WorldPoint camPosition = localPointToWorldTile(hdPlugin.camTarget[0], hdPlugin.camTarget[1]);
-		int camTargetX = camPosition.getX();
-		int camTargetY = camPosition.getY();
-		int camTargetZ = camPosition.getPlane();
+		WorldPoint camPosition = hdPlugin.getClient().getLocalPlayer().getWorldLocation();
```
[the remark this answers is not in the document]
Comment (Mark7625) [reply src/main/java/rs117/hd/environments/EnvironmentManager.java]: Thanks I have made the required changes

**Comment by sosodev** [comment]
It appears that this has some unintended effects. My house environment is different with the changes and that shouldn't be the case. 

house-branch
house-before

**Comment by sosodev** [comment]
With nothing defined it should just use the defaults I believe

**Pull request closed without merging**